test/e2e/apps/daemon_restart.go

```diff
@@ -318,4 +325,18 @@ var _ = SIGDescribe("DaemonRestart [Disruptive]", func() {
 			framework.Failf("Net container restart count went from %v -> %v after kubelet restart on nodes %v \n\n %+v", preRestarts, postRestarts, badNodes, tracker)
 		}
 	})
+
+	ginkgo.It("Kube-proxy should recover after being killed accidentally", func() {
```

Comment:
Did you also run this against a Linux node to make sure it passes?

Reply:
Yeah, just run it, it passed.